src/js/techreport/timeseries.js

```diff
@@ -184,17 +198,24 @@ class Timeseries {
         const latestSubcategory = latestEndpoint?.find(row => row.name === subcategory);
         const latestClient  = latestSubcategory?.[client];
         const latestValue = latestClient?.[metric];
+        const summaryValue = latestClient?.[summary];
 
         /* Select the container to which we'll add elements. */
         const card = container.querySelector(`[data-app="${app}"]`);
         const timestamp = component.querySelector('[data-slot="timestamp"]');
         const label = card.getElementsByClassName('breakdown-label')[0];
         const value = card.getElementsByClassName('breakdown-value')[0];
+        const secondaryValue = card.getElementsByClassName('breakdown-value-secondary')[0];
 
         /* Update text */
         label.innerHTML = latest.technology;
         if(latestValue) {
-          value.innerHTML = `${latestValue}${config.series.suffix || ''}`;
+          if(summary) {
+            value.innerHTML = `${summaryValue}`;
+            secondaryValue.innerHTML = `${latestValue}${config.series.suffix || ''}`;
+          } else {
+            value.innerHTML = `${latestValue}${config.series.suffix || ''}`;
+          }
         } else {
           value.classList.add('undefined');
           value.innerHTML = 'No data';
```

Comment:
Nit but we've recently discovered `innerHTML` (and `innerText`) is actually a bit slower than `textContent`.

It if really is just text (and I'm not sure if it always is), could we use `textContent`?

Reply:
So far it's only text, can update to `textContent`